### PR TITLE
fix(components): [tree-select] disabled attribute does not take effect

### LINKED ISSUE #10648: [Component] [tree-select] data绑定的ref值中有disabled:true属性，实际未禁用
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.41`
- Element Plus Version: `2.2.21`
- Browser / OS: `'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36'`
- Build Tool: `Vite`

### Related Component

- `el-tree-select`

### Reproduction Link

[JSFiddle](https://jsfiddle.net/Lsehf5p3/7/)

### Steps to reproduce

如代码，配置了props

### What is Expected?

正确的禁用选项

### What is actually happening?

未禁用

### Additional comments

希望可以补充props的完整性：
  其中value属性没有标出；
  disabled属性未与children属性保持默认一致，必须填

### PR DESCRIPTION
**Pull request by iDestin**
Add default values for attributes, fix #10648

closed #10648

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by iDestin** [comment]
can anyone help me review this? 🤓

**Review by tolking — APPROVED** [review]

**Comment by iDestin** [comment]
@yujinpan can you help me review this?

File: packages/components/virtual-list/__tests__/dynamic-size-list.test.ts [hunk 1]
```diff
@@ -117,9 +117,9 @@ describe('<dynamic-size-list />', () => {
         BASE_SIZE + 1 * 10
       )
 
-      expect(Number.parseInt(listRef.innerRef.style.height)).toBeLessThan(
-        BASE_SIZE + 5 * 12
-      )
+      expect(
+        Number.parseInt(listRef.innerRef.style.height)
+      ).toBeLessThanOrEqual(BASE_SIZE + 5 * 12)
```
Comment (yujinpan) [inline packages/components/virtual-list/__tests__/dynamic-size-list.test.ts:122]: Should not modify other unrelated code, no matter what the problem.

**Review by yujinpan — CHANGES_REQUESTED** [review]

**Comment by iDestin** [comment]
@yujinpan done PTAL

**Review by yujinpan — APPROVED** [review]
LGTM

**Pull request merged**